<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1385</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1385</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WI3DE Programm</string>
  </property>
  <property name="windowIcon">
   <iconset theme="икон">
    <normaloff>Images/икон.ico</normaloff>Images/икон.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>901</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Выберите фигуру, чтобы рассмотреть её 3D представление</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>1331</width>
      <height>254</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="img1">
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>211</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1211111</width>
             <height>1111112</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Images/cub2.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Cub">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>Куб</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="img1_2">
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>211</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1211111</width>
             <height>1111112</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Images/prizma.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Prizma">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>Призма</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="img1_3">
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>211</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1211111</width>
             <height>1111112</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Images/cilindr.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Cilindr">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>Цилиндр</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="img1_4">
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>211</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1211111</width>
             <height>1111112</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Images/piramida.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Piramida">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>Пирамида</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="img1_5">
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>211</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1211111</width>
             <height>1111112</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Images/shar.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Shar">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>Шар</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="img1_6">
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>211</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1211111</width>
             <height>1111112</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Images/konus2.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Konus">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>Конус</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="img1_7">
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>211</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>181</width>
           <height>1111112</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>Images/parall.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Konus_2">
         <property name="minimumSize">
          <size>
           <width>181</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>181</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>Параллелепипед</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Доброго времени суток, перед тем как воспользоваться программой, пожалуйста, ознокомьтесь с её управлением</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Ознакомиться с управлением</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>1421</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/Без имени-1.jpg</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_7</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
